src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_NUMBER = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_PROPERTY_VALUE = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final String defaultColor = Color.WHITE.name();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(FIGURE_NUMBER);
+
+        switch (index) {
+            case 1:
+                return getRandomCircle(getRandomProperty());
+            case 2:
+                return getRandomSquare(getRandomProperty());
+            case 3:
+                return getRandomRectangle(getRandomProperty(), getRandomProperty());
+            case 4:
```

Comment:
I think we could call getRandomProperty() inside methods, not pass as incoming parameter